docs/asset-registry/asset-registry-1.0.0.yaml

```
@@ -0,0 +1,710 @@
+            schema:
+              $ref: '#/components/schemas/BulkAssetCreateRequest'
+      responses:
+        '201':
+          description: Assets created successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BulkAssetCreateResponse'
+        '400':
+          description: Invalid input data
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)